src/assets/js/protocol/protocol-navigation.js

```diff
@@ -20,17 +20,19 @@ if (typeof Mzp === 'undefined') { // eslint-disable-line block-scoped-var
      * Event handler for navigation menu button `click` events.
      */
     Navigation.onClick = function(e) {
+        var thisNavItemList  = e.target.parentNode.querySelector('.mzp-c-navigation-items');
```

Comment:
If there can now be multiple navigation menus on one page, should we pass `thisNavItemList` as a parameter to `onNavOpen` and `onNavClose`, so those callbacks can work out which menu was interacted with?

Reply:
✅